calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -251,12 +248,15 @@ ModSecurity provides the following actions that you update in SecAction:
 
 #### Add or edit the rule set
 
-1. Create a directory, and download the core rules set files that you want to use.
+1. Create a directory, and download the core rules set files for customization.
 
 ```bash
 mkdir my-ruleset && cd my-ruleset
-curl -O https://raw.githubusercontent.com/coreruleset/coreruleset/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf
+curl -L https://github.com/coreruleset/coreruleset/archive/refs/heads/v3.4/dev.zip -o crs.zip
+unzip -j crs.zip 'coreruleset-3.4-dev/rules/*'
+rm crs.zip
 ```
+
 2. Download bootstrap files.
 
   Download the following required bootstrapping configuration files [Reference](https://medium.com/lightbaseio/web-application-firewall-in-go-feat-owasp-modsecurity-core-rule-set-3f97a26e3311) that are used by Tigera operator:
```

Comment:
Download the following required bootstrapping configuration files from this [site]( https://medium.com/lightbaseio/web-application-firewall-in-go-feat-owasp-modsecurity-core-rule-set-3f97a26e3311). These files are used by Tigera operator.